packages/java-shell/settings.gradle

```diff
@@ -4,5 +4,5 @@ pluginManagement {
         maven { url 'https://plugins.gradle.org/m2/' }
     }
 }
-rootProject.name = 'java-shell'
+rootProject.name = 'JMongosh'
```

Comment:
Just a note, our PR titles refer to the individual directories inside the `packages/` folder, so if you want to start using `feat(jmongosh):`, you will also want to rename the directory (and then run `npm run update-evergreen-config` in the top-level directory of this repo)